service/controller/app/v1/resource/tcnamespace/create.go

```diff
@@ -32,6 +32,14 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return nil
 	}
 
+	// Resource is used to bootstrap chart-operator in tenant clusters.
+	// So for other apps we can skip this step.
```

Comment:
The `giantswarm` namespace only needs to be created for the chart-operator app. As this is part of the bootstrap process.